include/package.mk

```diff
@@ -323,10 +323,7 @@ Build/Install=$(if $(PKG_INSTALL),$(call Build/Install/Default,))
 Build/Dist=$(call Build/Dist/Default,)
 Build/DistCheck=$(call Build/DistCheck/Default,)
 
-.NOTPARALLEL:
```

Comment:
at first I was moving it around, removal is a mistake

I guess I would just have to ask you how it works because i thought (possibly wrongly) that if a target has a full empty line under it then theres nothing in it

 every example I find with a quick search shows targets listed after the colon of special targets like they are prerequisites of a normal target

so if that's not true how does it work? does it include every target under it or just the first one under it?

Reply:
If it's placed anywhere within the makefile, it applies to the whole make context